internal/server/v0/internalplace/golden/get_place_page_data/asm.sample.json

```diff
@@ -25629,6 +25629,14 @@
       }
     }
   },
+  "validCategories": {
```

Comment:
Why are these all empty?

Reply:
Because we don't have a category in the request for these test cases. So this uses the old cache. The file ca_economics.sample.json has a category included and we can see desired behavior in that file.

Reply:
That makes sense, thanks!